cartridges/app_custom_pd/cartridge/templates/default/experience/components/commerce_assets/categoryComponent.isml

```diff
@@ -0,0 +1,35 @@
+<isscript>
+    var assets = require('*/cartridge/scripts/assets.js');
+    assets.addCss('/css/experience/components/commerceAssets/popularCategory.css');
+    assets.addCss('/css/experience/components/commerceAssets/categoryComponent.css');
+</isscript>
+
+<div class="mt-2 tile-container">
+    <div class="image-cropper">
+        <a href="${pdict.category.url}">
+            <isif condition="${pdict.category.customImageUrl}">
+                <picture>
```

Comment:
Why do we use picture tag as we don't have different sources per breakpoint

Reply:
yes, i don`t have to use picture tag in that case